Dockerfile

```diff
@@ -19,18 +19,15 @@ RUN apt-get update \
 RUN python3 -m venv /opt/relay
 
 WORKDIR /relay
-COPY ./constraints.txt /relay/constraints.txt
-RUN /opt/relay/bin/pip install --disable-pip-version-check -c constraints.txt pip wheel setuptools
+COPY ./dev-requirements.txt /relay/dev-requirements.txt
```

Comment:
Is there a reason to use `dev-requirements` instead of non-dev `requirements`?

Reply:
This is for pip, wheel and setuptools, which we currently have defined as development requirements. I guess we should change that then, mmh